elements/pf-tabs/test/pf-tabs.spec.ts

```diff
@@ -141,34 +140,40 @@ describe('<pf-tabs>', function() {
   });
 
   describe('disabled tabs', function() {
```

Comment:
we can say that these tests are written as
"assert result when predicate obtains"

we should generally prefer to factor tests as
"given a predicate, assert result"

meaning, use nested `describe` blocks with `beforeEach` to set up state, and `it` should only check resulting state. see the rh-table pr, other examples in the repo.

Reply:
I'll take a look at a refactor